cpp/src/io/functions.cpp

```diff
@@ -123,15 +123,13 @@ namespace {
 
 std::vector<std::unique_ptr<cudf::io::datasource>> make_datasources(source_info const& info,
                                                                     size_t offset            = 0,
-                                                                    size_t max_size_estimate = 0,
-                                                                    size_t min_size_estimate = 0)
+                                                                    size_t max_size_estimate = 0)
```

Comment:
Removing `min_size_estimate` should make this PR have a `breaking` label. Is that correct? @vuule

Reply:
That's correct, even though the breakage doesn't affect any users.